packages/launchpad/src/Main.vue

```diff
@@ -3,7 +3,17 @@
     <HeaderBar
       class="w-full z-10 fixed"
     />
-    <div class="px-24px pt-86px">
+
+    <MigrationLandingPage
+      v-if="currentProject?.needsLegacyConfigMigration && !wasLandingPageShown && online && videoHtml"
```

Comment:
If the user has an error (like not having Cypress installed int he project), it will be shown after they continue pass this page. Not sure if it's a better experience throwing the error sooner, but it felt like "Ah yeah, Cypress 10 looks dope" then boom error.

Reply:
I guess let's see how it feels in the final run through, we could tweak